c2corg_ui/templates/index.html

```diff
@@ -38,4 +38,20 @@ <h1 class="welcome-title text-center" translate>The mountain sports community</h
     </div>
   </section>
   <app-feed id="feed"></app-feed>
+
+  <div class="float-buttons" ng-if="userCtrl.auth.isAuthenticated()">
+    <div class="float-button-container">
+      <div class="float-button" ng-click="feedCtrl.toggleFilters()" ng-class="{'personal-feed-on': feedCtrl.isPersonal}">
+        <span class="glyphicon glyphicon-user"></span>
+      </div>
+      <p class="float-button-text" x-translate ng-bind="feedCtrl.isPersonal ? 'Personal feed on' : 'Personal feed off'">Personal feed</p>
```

Comment:
What about my suggestion with the translate filters? 
Such a solution has already been used for instance here:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/edit.html#L29

Adding translations to ``i18n.html`` should be avoided as much as possible because if you change or remove the strings in the code, you have to make sure they are also updated in this template.